src/UI/Areas/Identity/Pages/Account/Manage/ManageNavPages.cs

```diff
@@ -12,10 +12,6 @@ public static class ManageNavPages
 
         public static string ChangePassword => "ChangePassword";
 
-        public static string DownloadPersonalData => "DownloadPersonalData";
```

Comment:
The changes look good. The only thing  we need to change are the removed members and the visibility changes. Is there any issue if you remove those changes? Those are breaking changes and we would ideally like to take this into one of our patch releases, so we need to revert those changes even if the methods are not used.

Reply:
@javiercn Yeah, my thought process at the time was that these changes shouldn't break anything, as it's scaffolded code. But now that you mention it, if someone were to have scaffolded one of the pages that was changed, and then updates Identity UI with these changes, it's likely to break something.

I'm not sure how you want me to go about this. The breaking changes are in two commits, so they can easily be reverted. Should the breaking changes be reverted and put into a separate pull request?

Reply:
@tstivers1990 I think scraping them all together is fine. While it's not super clean, its in a .Internal namespace so it doesn't really matter. I don't want to trouble you into sending another PR as there's not much value on fixing it. Scraping the commits with the breaking changes will work fine.

Thanks for your contribution